pkg/odo/cli/delete/component.go

```diff
@@ -42,6 +63,28 @@ func (o *ComponentOptions) Run() error {
 
 // deleteNamedComponent deletes a component given its name
 func (o *ComponentOptions) deleteNamedComponent() error {
+	log.Info("Searching resources to delete, please wait...")
+	list, err := o.clientset.DeleteClient.ListResourcesToDelete(o.name, o.clientset.KubernetesClient.GetCurrentNamespace())
```

Comment:
Maybe add a spinner here for better user experience?

Reply:
I would prefer to keep the output as simple as possible, and let's @cdrage work on these output as mentioned in https://kubernetes.slack.com/archives/C01D6L2NUAG/p1646834639864189?thread_ts=1646810843.506709&cid=C01D6L2NUAG